dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/cases/TenantAPITest.java

```diff
@@ -76,38 +76,9 @@ public void testCreateTenant() {
     public void testDuplicateCreateTenant() {
         TenantPage tenantPage = new TenantPage();
 
-        HttpResponse createTenantHttpResponse = tenantPage.createTenant(sessionId, tenant, 1, "");
+        HttpResponse createTenantHttpResponse = tenantPage.createTenant(sessionId, tenantName, 1, "");
 
         Assertions.assertFalse(createTenantHttpResponse.body().success());
     }
 
-    @Test
-    @Order(5)
-    public void testGetTenantListPaging() {
-        TenantPage tenantPage = new TenantPage();
-
-        HttpResponse createTenantHttpResponse = tenantPage.getTenantListPaging(sessionId, 1, 10, "");
-        boolean result = false;
-
-        for (TenantListPagingResponseTotalList tenantListPagingResponseTotalList : JSONUtils.convertValue(createTenantHttpResponse.body().data(), TenantListPagingResponseData.class).totalList()) {
-            if (tenantListPagingResponseTotalList.tenantCode().equals(tenant)) {
-                result = true;
-                existTenantId = tenantListPagingResponseTotalList.id();
-                break;
-            }
-        }
-
-        Assertions.assertTrue(createTenantHttpResponse.body().success());
-        Assertions.assertTrue(result);
-    }
-
-    @Test
-    @Order(10)
-    public void testDeleteTenant() {
-        TenantPage tenantPage = new TenantPage();
-
-        HttpResponse deleteTenantHttpResponse = tenantPage.deleteTenant(sessionId, existTenantId);
-
-        Assertions.assertTrue(deleteTenantHttpResponse.body().success());
-    }
```

Comment:
Is there any reason for why we need to delete these tests?

Reply:
> Is there any reason for why we need to delete these tests?

Currently checkstyle not enabled in api-test...

Reply:
I think starting checkstyle at the beginning is a good start. I'll enable checkstyle in api-test these days.

Reply:
API-Test's checkstyle have been enabled in #10805.

Reply:
Why did you delete these?

Reply:
This has not been addressed.